templates/zerver/help/connect-through-a-proxy.md

```diff
@@ -0,0 +1,77 @@
+# Connect through a proxy
+
+Some corporate and university networks may require you to connect to Zulip
+via a proxy.
+
+## Web
+
+Zulip uses your browser's default proxy settings. To set a custom proxy just
+for Zulip, check your browser's instructions for setting a custom proxy for
+a single website.
+
+## Desktop
+
+1. Click on the **gear** (<i class="icon-vector-cog"></i>) icon in the lower left corner.
+
+2. Select the **Network** tab.
+
+### System proxy settings
+
+3. Click **Use system proxy**.
+
+4. Restart the Zulip desktop app.
+
+### Custom proxy settings
+
+3. Click **Custom proxy**.
+
+4. Either enter a URL for **PAC script**, or fill out **Proxy rules** and
+  **Proxy bypass rules**.
+
+5. Click **Save changes**.
+
+In most corporate environments, your network administrator will provide a
+URL for the **PAC script**.
+
+The second most common configuration is that your network administrator has
+set up a proxy server for accessing the public internet, but URLs on the
+local network must be accessed directly. In that case set **Proxy rules** to
+the URL of the proxy server (it may look something like
+`proxy.example.edu:port`), and **Proxy bypass rules** to cover local URLs
+(it may look something like `*.example.edu;10.0.0.0/8`).
```

Comment:
As you mentioned, proxy bypass rules format doesn't have any semicolon format, I checked the correct usage. It is a comma separated list, so `*.example.edu,10.0.0.0/8` should be correct.